### PR TITLE
Add note for app installation about RSC apps

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

File: concepts/known-issues.md [hunk 1]
```diff
@@ -408,6 +408,9 @@ In certain instances, the `tenantId` / `email` / `displayName` property for the
 ### Properties are missing in the list of teams that a user has joined
 The API call for [me/joinedTeams](/graph/api/user-list-joinedteams) returns only the **id**, **displayName**, and **description** properties of a [team](/graph/api/resources/team). To get all properties, use the [Get team](/graph/api/team-get) operation.
 
+### Installation of apps that require resource-specific consent permissions is not supported
+The API calls for [add app to team](/graph/api/team-post-installedapps.md), [add app to chat](/graph/api/chat-post-installedapps.md), and [add app for user](/graph/api/userteamwork-post-installedapps.md) do not support installing apps that require [resource-specific consent](https://aka.ms/teams-rsc) permissions.
```
Comment (AkJo) [inline concepts/known-issues.md:412]: We need to call out the respective upgrade APIs as well.

File: concepts/known-issues.md [hunk 2]
```diff
@@ -408,6 +408,15 @@ In certain instances, the `tenantId` / `email` / `displayName` property for the
 ### Properties are missing in the list of teams that a user has joined
 The API call for [me/joinedTeams](/graph/api/user-list-joinedteams) returns only the **id**, **displayName**, and **description** properties of a [team](/graph/api/resources/team). To get all properties, use the [Get team](/graph/api/team-get) operation.
 
+### Installation of apps that require resource-specific consent permissions is not supported
+The following API calls do not support installing apps that require [resource-specific consent](https://aka.ms/teams-rsc) permissions.
+- [Add app to team](/graph/api/team-post-installedapps.md)
+- [Upgrade app installed in team](/graph/api/team-teamsappinstallation-upgrade.md)
+- [Add app to chat](/graph/api/chat-post-installedapps.md)
+- [Upgrade app installed in chat](/graph/api/chat-teamsappinstallation-upgrade.md)
+- [Add app for user](/graph/api/userteamwork-post-installedapps.md)
```
Comment (AkJo) [inline concepts/known-issues.md:417]: Remove these two that pertain to user scoped apps.

**Review by AkJo — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/chat-post-installedapps.md [hunk 3]
```diff
@@ -15,7 +15,9 @@ Namespace: microsoft.graph
 
 Install a [teamsApp](../resources/teamsapp.md) to the specified [chat](../resources/chat.md).
 
-> **Note**: If the chat is associated with an [onlineMeeting](../resources/onlinemeeting.md) instance, then, effectively, the **teamsApp** will get installed to the meeting.
+> **Notes**:
+> - If the chat is associated with an [onlineMeeting](../resources/onlinemeeting.md) instance, then, effectively, the **teamsApp** will get installed to the meeting.
+> - Currently, this operation does not support installation of apps that require resource-specific consent permissions. For details, see [Known issues](/graph/known-issues#Installation-of-apps-that-require-resource-specific-consent-permissions-is-not-supported).
```
Comment (AkJo) [inline api-reference/beta/api/chat-post-installedapps.md:20]: @Lauragra , should both "K" and "I" in Known Issues be uppercase, or should both be lower case?

File: api-reference/beta/api/chat-teamsappinstallation-upgrade.md [hunk 4]
```diff
@@ -15,7 +15,9 @@ Namespace: microsoft.graph
 
 Upgrade an [app installation](../resources/teamsappinstallation.md) within a [chat](../resources/chat.md).
 
-> **Note**: If the chat is associated with an [onlineMeeting](../resources/onlinemeeting.md) instance, then effectively, the **teamsApp** installed in the meeting will get upgraded.
+> **Notes**:
+> - If the chat is associated with an [onlineMeeting](../resources/onlinemeeting.md) instance, then effectively, the **teamsApp** installed in the meeting will get upgraded.
+> - Currently, this operation does not support upgrade of apps that require resource-specific consent permissions. For details, see [Known issues](/graph/known-issues#Installation-of-apps-that-require-resource-specific-consent-permissions-is-not-supported).
```
Comment (AkJo) [inline api-reference/beta/api/chat-teamsappinstallation-upgrade.md:20]: Same comment applies here and elsewhere. Please gets @Lauragra 's guidance.

File: api-reference/beta/api/team-teamsappinstallation-upgrade.md [hunk 5]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 Upgrade an [app installation](../resources/teamsappinstallation.md) in a [team](../resources/team.md) to the latest version of the app.
 
+> **Note:** Currently, this operation does not support upgrade of apps that require resource-specific consent permissions. For details, see [Known issues](/graph/known-issues#Installation-of-apps-that-require-resource-specific-consent-permissions-is-not-supported).
```
Comment (AkJo) [inline api-reference/beta/api/team-teamsappinstallation-upgrade.md:18]: Please do ensure that the hyperlink does take you to the right section when the changes are live.

**Review by AkJo — APPROVED** [review]
Approved with comments.

File: api-reference/beta/api/chat-teamsappinstallation-upgrade.md (hunk 4 above)
Comment (Lauragra) [reply api-reference/beta/api/chat-teamsappinstallation-upgrade.md:20]: This is correct - Known issues (sentence case).

File: api-reference/beta/api/chat-post-installedapps.md (hunk 3 above)
Comment (Lauragra) [reply api-reference/beta/api/chat-post-installedapps.md:20]: We use sentence case for titles so Known issues is correct.

File: api-reference/beta/api/team-teamsappinstallation-upgrade.md (hunk 5 above)
Comment (jessiechan27) [reply api-reference/beta/api/team-teamsappinstallation-upgrade.md:18]: Will do

**Pull request merged**